source/index.erb

```diff
@@ -4,17 +4,23 @@ title: Energy Globe Award
 
 <header class="page-header">
   <div class="container">
-    <div class="content">
-      <h1 class="headline">Finde ausgezeichnete nachhaltige Projekte</h1>
+    <h1 class="title">Energy Globe Award Gewinner</h1>
+    <h2 class="headline">Die besten Projekte des Global Energy Award auf einem Blick.</h2>
+  </div>
+  <img class="background" src="/images/big-header-fading.jpg" />
```

Comment:
Because that doesn't work the same way – I actually started with a background image. But you can't tell a background image to be at least as big as the element but never smaller than the original size of the image :wink: this is exactly why i used `min-width` to answer your question below. You can think of it as like a "softer" form of `width: 100%`